helpers/infer.js

```diff
@@ -114,7 +164,7 @@ export const inferVersionInfo = async function (rawFile, project, gameVersions)
         loaders: ['fabric'],
         version_type: versionType(metadata.version),
         game_versions: metadata.depends
-          ? gameVersionRange(metadata.depends.minecraft, gameVersions)
+          ? getGameVersionsMatchingSemverRange(metadata.depends.minecraft, simplifiedGameVersions)
```

Comment:
Looks good for the most part, but you forgot to apply this to quilt.mod.jsons too

Reply:
the second line of the `getGameVersionsMatchingSemverRange` checks if the input is a single range or an array of ranges and does it accordingly